i18n/fr/docusaurus-plugin-content-docs/version-23.04/service-mapping/install.md

```diff
@@ -12,7 +12,9 @@ import TabItem from '@theme/TabItem';
 
 ## Prerequisites
 
-Le serveur central et Centreon BAM doivent être dans la même version majeure (c'est-à-dire tous les deux en 22.10.x).
+Installez BAM sur le serveur central.
+Le serveur central et Centreon BAM doivent être dans la même version majeure (c'est-à-dire tous les deux en 23.04.x).
```

Comment:
It's necessary to specify 23.04.x here ?

Reply:
it's clearer